modules/bitgo/test/v2/unit/tss/helpers.ts

```diff
@@ -19,3 +20,28 @@ export async function nockGetTxRequest(params: {walletId: string, txRequestId: s
     .get(`/api/v2/wallet/${params.walletId}/txrequests?txRequestIds=${params.txRequestId}&latest=true`)
     .reply(200, params.response);
 }
+
+export async function createWalletSignatures(
+  privateKeyArmored: string,
+  publicKeyToCertArmoredUser: string,
+  publicKeyToCertArmoredBackup: string,
+  notations: { name: string; value: string }[]
+): Promise<string> {
+  const certsUserString = await createSharedDataProof(privateKeyArmored, publicKeyToCertArmoredUser, notations);
```

Comment:
nit: if these are armored strings, then rename to something like "userWalletSigArmored`

Reply:
Renamed